components/script/dom/webglrenderingcontext.rs

```diff
@@ -2345,6 +2345,41 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
         Int32Value(receiver.recv().unwrap())
     }
 
+    #[allow(unsafe_code)]
+    // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.7
+    unsafe fn GetRenderbufferParameter(&self, _cx: *mut JSContext, target: u32, pname: u32) -> JSVal {
+        let target_matches = target == constants::RENDERBUFFER;
+
+        let pname_matches = match pname {
+            constants::RENDERBUFFER_WIDTH |
+            constants::RENDERBUFFER_HEIGHT |
+            constants::RENDERBUFFER_INTERNAL_FORMAT |
+            constants::RENDERBUFFER_RED_SIZE |
+            constants::RENDERBUFFER_GREEN_SIZE |
+            constants::RENDERBUFFER_BLUE_SIZE |
+            constants::RENDERBUFFER_ALPHA_SIZE |
+            constants::RENDERBUFFER_DEPTH_SIZE |
+            constants::RENDERBUFFER_STENCIL_SIZE => true,
+            _ => false,
+        };
+
+        if !target_matches || !pname_matches {
+            self.webgl_error(InvalidEnum);
+            return NullValue();
+        }
+
+        let (sender, receiver) = webgl_channel().unwrap();
+        self.send_command(WebGLCommand::GetRenderbufferParameter(target, pname, sender));
+
```

Comment:
The following check is missing:
`If  the  renderbuffer  currently  bound  to
target
is  zero,   then
INVALID_-
OPERATION
is generated.`

Reply:
It's just this, basically :)
```Rust
if self.bound_renderbuffer.get().is_none() {
    self.webgl_error(InvalidOperation);
    return NullValue();
}
```